main.cpp

```diff
@@ -564,9 +637,11 @@ struct GameStore
     GameStore();
     ~GameStore();
 
-    void sellControllersWithXbox(Xbox, int numControllersToSell);
-    void displayGame(Xbox::Game gameA);
+    void sellControllersWithXbox(Xbox&, int numControllersToSell);
```

Comment:
pass by `const reference` if you're not modifying the passed-in object
- [ ] revise